<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tak Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color:rgb(211, 215, 207)</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>745</width>
         <height>540</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QTextEdit" name="textEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
          <property name="overwriteMode">
           <bool>true</bool>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextEditable|Qt::TextEditorInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>765</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="actionPrint_PDF"/>
    <addaction name="actionPrint_Export"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Format</string>
    </property>
    <addaction name="actionFont"/>
    <addaction name="actionColor"/>
   </widget>
   <widget class="QMenu" name="menuSettinngs">
    <property name="title">
     <string>Style</string>
    </property>
    <addaction name="actionBold"/>
    <addaction name="actionItalic"/>
    <addaction name="actionUnderline"/>
    <addaction name="separator"/>
    <addaction name="actionCenter_Align"/>
    <addaction name="actionLeft_Align"/>
    <addaction name="actionRight_Align"/>
    <addaction name="actionJustify"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Time And Date</string>
    </property>
    <addaction name="actionTime"/>
    <addaction name="actionDate"/>
   </widget>
   <widget class="QMenu" name="menuHelp_2">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuSettinngs"/>
   <addaction name="menuWindows"/>
   <addaction name="menuHelp_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionPrint"/>
   <addaction name="actionPrint_Export"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionFont"/>
   <addaction name="actionColor"/>
   <addaction name="separator"/>
   <addaction name="actionBold"/>
   <addaction name="actionItalic"/>
   <addaction name="actionUnderline"/>
   <addaction name="separator"/>
   <addaction name="actionLeft_Align"/>
   <addaction name="actionCenter_Align"/>
   <addaction name="actionRight_Align"/>
   <addaction name="actionJustify"/>
   <addaction name="separator"/>
   <addaction name="actionTime"/>
   <addaction name="actionDate"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="items/itemsresource.qrc">
     <normaloff>:/items/open-folder.png</normaloff>:/items/open-folder.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="items/itemsresource.qrc">
     <normaloff>:/items/diskette.png</normaloff>:/items/diskette.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="items/itemsresource.qrc">
     <normaloff>:/items/diskette.png</normaloff>:/items/diskette.png</iconset>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="items/itemsresource.qrc">
     <normaloff>:/items/printer.png</normaloff>:/items/printer.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionPrint_PDF">
   <property name="icon">
    <iconset resource="items/itemsresource.qrc">
     <normaloff>:/items/printer.png</normaloff>:/items/printer.png</iconset>
   </property>
   <property name="text">
    <string>Print PDF</string>
   </property>
  </action>
  <action name="actionPrint_Export">
   <property name="icon">
    <iconset resource="items/itemsresource.qrc">
     <normaloff>:/items/pdf.png</normaloff>:/items/pdf.png</iconset>
   </property>
   <property name="text">
    <string>Export PDF</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="items/itemsresource.qrc">
     <normaloff>:/items/copy.png</normaloff>:/items/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="items/itemsresource.qrc">
     <normaloff>:/items/cut.png</normaloff>:/items/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="items/itemsresource.qrc">
     <normaloff>:/items/paste.png</normaloff>:/items/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="items/itemsresource.qrc">
     <normaloff>:/items/undo.png</normaloff>:/items/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="items/itemsresource.qrc">
     <normaloff>:/items/redo.png</normaloff>:/items/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset resource="items/itemsresource.qrc">
     <normaloff>:/items/iconfinder_text_103723.png</normaloff>:/items/iconfinder_text_103723.png</iconset>
   </property>
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="icon">
    <iconset resource="items/itemsresource.qrc">
     <normaloff>:/items/iconfinder_Color_Board_381595.png</normaloff>:/items/iconfinder_Color_Board_381595.png</iconset>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="icon">
    <iconset resource="items/itemsresource.qrc">
     <normaloff>:/items/iconfinder_ic_format_bold_48px_352381.png</normaloff>:/items/iconfinder_ic_format_bold_48px_352381.png</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="icon">
    <iconset resource="items/itemsresource.qrc">
     <normaloff>:/items/iconfinder_ic_format_italic_48px_352387.png</normaloff>:/items/iconfinder_ic_format_italic_48px_352387.png</iconset>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="icon">
    <iconset resource="items/itemsresource.qrc">
     <normaloff>:/items/iconfinder_text-format_326710.png</normaloff>:/items/iconfinder_text-format_326710.png</iconset>
   </property>
   <property name="text">
    <string>Underline</string>
   </property>
  </action>
  <action name="actionCenter_Align">
   <property name="icon">
    <iconset resource="items/itemsresource.qrc">
     <normaloff>:/items/iconfinder_align-center_3324978.png</normaloff>:/items/iconfinder_align-center_3324978.png</iconset>
   </property>
   <property name="text">
    <string>Left Align</string>
   </property>
  </action>
  <action name="actionLeft_Align">
   <property name="icon">
    <iconset resource="items/itemsresource.qrc">
     <normaloff>:/items/iconfinder_align-left_2561323.png</normaloff>:/items/iconfinder_align-left_2561323.png</iconset>
   </property>
   <property name="text">
    <string>Center Align</string>
   </property>
  </action>
  <action name="actionRight_Align">
   <property name="icon">
    <iconset resource="items/itemsresource.qrc">
     <normaloff>:/items/iconfinder_align-right_2561324.png</normaloff>:/items/iconfinder_align-right_2561324.png</iconset>
   </property>
   <property name="text">
    <string>Right Align</string>
   </property>
  </action>
  <action name="actionJustify">
   <property name="icon">
    <iconset resource="items/itemsresource.qrc">
     <normaloff>:/items/iconfinder_align-justify_3324979.png</normaloff>:/items/iconfinder_align-justify_3324979.png</iconset>
   </property>
   <property name="text">
    <string>Justify</string>
   </property>
  </action>
  <action name="actionTime">
   <property name="icon">
    <iconset resource="items/itemsresource.qrc">
     <normaloff>:/items/iconfinder_document-09_1622827.png</normaloff>:/items/iconfinder_document-09_1622827.png</iconset>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
  </action>
  <action name="actionDate">
   <property name="icon">
    <iconset resource="items/itemsresource.qrc">
     <normaloff>:/items/iconfinder_document-08_1622828.png</normaloff>:/items/iconfinder_document-08_1622828.png</iconset>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="items/itemsresource.qrc">
     <normaloff>:/items/addition.png</normaloff>:/items/addition.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="items/itemsresource.qrc"/>
 </resources>
 <connections/>
</ui>
